test/SourceKit/Sema/sema_diag_after_edit_fixit.swift

```diff
@@ -4,7 +4,7 @@ let a = 0; let b = 0 }; unresolved
 // Test that offsets of diagnostic ranges and fixits get updated correctly after the edit request
 
 // RUN: %sourcekitd-test -req=open %s -- %s == -req=print-diags %s \
-// RUN:    == -req=edit -pos=2:1 -replace="_" -length=5 %s -print-raw-response \
+// RUN:    == -req=edit -pos=2:1 -replace="_" -length=5 %s == -req=print-diags %s \
```

Comment:
Can you explain what happened here?  I think I understand all the other test changes, but not clear on this one.